File: Joon/JS/baekjoon/9093_단어뒤집기.js
const input = require("fs")
  .readFileSync("/dev/stdin")
  .toString()
  .trim()
  .split("\n");

const t = Number(input[0]);

function reverse(string) {
  let stack = [];
  let answer = "";
  for (let i = 0; i < string.length; i++) {
    stack.push(string[i]);
  }

  for (let i = 0; i < string.length; i++) {
    answer += stack.pop();
  }

  return answer;
}

let result = [];
for (let i = 1; i < input.length; i++) {
  result.push(input[i].split(" ").map(reverse));
}

for (let i = 0; i < result.length; i++) {
  console.log(result[i].join(" "));
}

File: Joon/Python/baekjoon/9093_단어뒤집기.py
import sys

def reverse(string):
    result = ""
    stack = []
    for char in string:
        stack.append(char)
    
    for i in range(len(string)):
        result += stack.pop()
    
    return result
n = int(sys.stdin.readline())

for i in range(n):
    result = []
    input = sys.stdin.readline().strip().split();
    for i in range(len(input)):
        result.append(reverse(input[i]))
    print(" ".join(result))


